pylzy/tests/scenarios/custom_serializer/__init__.py

```diff
@@ -0,0 +1,24 @@
+import uuid
+
+from lzy.api.v1 import Lzy, op
+from serializer import TestStrSerializer
+
+
+class Kek:
+    pass
+
+
+@op
+def func(s: str) -> str:
+    print(s)
+    return "some_string"
+
+
+WORKFLOW_NAME = "workflow_" + str(uuid.uuid4())
+
+if __name__ == "__main__":
+    lzy = Lzy()
+    lzy.serializer_registry.register_serializer(TestStrSerializer())
```

Comment:
maybe `serializerS_registry`?

Reply:
Probably no: 
https://github.com/lambdazy/lzy/pull/1115/files#diff-f94fed5076ba023618ec9857b46ee67af409abd5b28daaabb2ea92d06f8c3b48L188

At least not in this PR